lib/liquid/mixed.rb

```diff
@@ -0,0 +1,13 @@
+module Liquid
+  module Mixed
+    def present(input)
+      input.present?
+    end
+
+    def blank(input)
+      input.blank?
```

Comment:
These methods aren't defined in ruby, so this is implicitly dependent on active support.